relayer/chains/substrate/event_parser.go

```diff
@@ -244,26 +259,53 @@ func (pkt *packetInfo) MarshalLogObject(enc zapcore.ObjectEncoder) error {
 func (pkt *packetInfo) parseAttrs(log *zap.Logger, attributes interface{}) {
 	attrs := attributes.(ibcEventQueryItem)
 
-	pkt.Sequence = attrs["sequence"].(uint64)
+	var err error
+
+	var height exported.Height
+	if height, err = parseHeight(attrs["height"]); err != nil {
+		log.Error("error parsing connection height: ",
+			zap.Error(err),
+		)
+		return
+	}
+	pkt.Height = height.GetRevisionHeight()
+
+	pkt.Sequence = cast.ToUint64(attrs["sequence"].(float64))
 	pkt.SourcePort = attrs["source_port"].(string)
 	pkt.SourceChannel = attrs["source_channel"].(string)
 	pkt.DestPort = attrs["destination_port"].(string)
 	pkt.DestChannel = attrs["destination_channel"].(string)
-	pkt.Data = attrs["data"].([]byte)
 
-	var err error
+	if pkt.Data, err = rpcclienttypes.HexDecodeString(attrs["data"].(string)); err != nil {
+		log.Error("error parsing packet data: ",
+			zap.Error(err),
+		)
+		return
+	}
+
 	if pkt.TimeoutHeight, err = parseHeight(attrs["timeout_height"]); err != nil {
 		log.Error("error parsing packet height: ",
 			zap.Error(err),
 		)
 		return
 	}
 
-	pkt.TimeoutTimestamp = attrs["timeout_timestamp"].(uint64)
+	if pkt.TimeoutTimestamp, err = parseTimestamp(attrs["timeout_timestamp"]); err != nil {
+		log.Error("error parsing packet timeout timestamp: ",
```

Comment:
Logs should begin with a capital letter and not end in punctuation.

See https://github.com/cosmos/relayer/blob/main/docs/logging.md